src/thimble_server/data_access_layer/s3/store.clj

```diff
@@ -0,0 +1,10 @@
+;; TODO: fns to:
+;;             generate safe filename
+;;             save to s3
+;;             take audio and call other fns and return filename [or something else]
+
+;; just returns static string at the moment for testing other logics
+(defn store-audio!
+  "takes audio, stores to s3 returns filename-key"
```

Comment:
i'd prefer we don't name this something about s3, as something importing this namespace doesn't need to know that detail.

i'd just put this in data_access_layer/file_storage.clj